src/Microsoft.OData.Core/JsonLight/ODataJsonLightOutputContext.cs

```diff
@@ -132,7 +132,8 @@ internal sealed class ODataJsonLightOutputContext : ODataOutputContext
             Debug.Assert(messageWriterSettings != null, "messageWriterSettings != null");
 
             this.textWriter = textWriter;
-            this.jsonWriter = CreateJsonWriter(messageInfo.Container, textWriter, true /*isIeee754Compatible*/, messageWriterSettings);
+            bool ieee754CompatibleSetToTrue = (messageInfo.MediaType != null) ? messageInfo.MediaType.HasIeee754CompatibleSetToTrue() : false;

```

Comment:
>: false; [](start = 134, length = 8)

default as "true" for non-regression?

Reply:
how about this?

---
In reply to: [375393605](https://github.com/OData/odata.net/pull/1659#discussion_r375393605) [](ancestors = 375393605)